README.md

```diff
@@ -61,8 +61,307 @@ Your support is greatly appreciated. Development of this free, open-source proje
 
 - Environment (pick one)
   - Python 3.10 or later (instructions: [for Windows](https://www.tutorialspoint.com/how-to-install-python-in-windows))
```

Comment:
Merge went sideways for this file

Reply:
Try and put the docs in the correct file. We split up the readme to make it less of a block of test

Reply:
Fixed, thanks.